prow/config.yaml

```diff
@@ -353,7 +456,54 @@ presubmits:
     skip_branches:
     - release-1.4
     - release-1.5
-    - release-1.6  # doesn't have BUILD files under //vendor/k8s.io
+    - release-1.6 # doesn't have BUILD files under //vendor/k8s.io
+    - release-1.7 # different set of targets
+    - release-1.8 # different set of targets
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/bazelbuild:v20171005-2f37588f-0.6.1
+        args:
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--" # end bootstrap args, scenario args below
+        - "--test=//..."
```

Comment:
similarly `make bazel-test` is `//... //hack:verify-all -//build/... -//vendor/...`

Reply:
I'll update this once we have negative deps in.

Reply:
done